styles.css

```diff
@@ -1,98 +1,134 @@
 }
 
 .logo__circle {
-              position: absolute;
-              left: 0;
-              top: 50%;
-              transform: translateY(-50%);
-              width: 28px;
-              height: 28px;
-              border-radius: 50%;
-              opacity: .7;
+  position: absolute;
+  left: 0;
+  top: 50%;
+  transform: translateY(-50%);
+  width: 28px;
+  height: 28px;
+  border-radius: 50%;
+  opacity: .7;
 }
 
 .logo__circle_color_dark-blue{
-                             background: #0056EC;
+  background: #0056EC;
 }
 
 .logo__circle_color_blue{
-                        left: 8px;
-                        top: 56%;
-                        background: #C8DBFB;
+  left: 8px;
+  top: 56%;
+  background: #C8DBFB;
 }
 
 .menu {
-      display: flex;
-      align-items: center;
+  display: flex;
+  align-items: center;
 }
 
 .menu__item {
-            position: relative;
-            margin-right: 26px;
-            font: 500 18px/21px robotomedium, arial, helvetica, sans-serif;
-            color: #000;
-            text-decoration: none;
-            transition: opacity .5s;
+  position: relative;
+  margin-right: 27px;
+  font: 500 18px/21px robotomedium, arial, helvetica, sans-serif;
+  color: #000;
+  text-decoration: none;
+  transition: opacity .5s;
 }
 
 .menu__item:last-child {
-                       margin-right: 0;
+  margin-right: 10px;
 }
 
 .menu__item_active::after {
-                          position: absolute;
-                          display: block;
-                          content: '';
-                          left: 0;
-                          width: 100%;
-                          height: 2px;
-                          background: #0056EC;
+  position: absolute;
+  display: block;
+  content: '';
+  left: 0;
+  width: 100%;
+  height: 2px;
+  background: #0056EC;
 }
 
 .menu__item:after{
-                  position: absolute;
-                  display: block;
-                  content: '';
-                  left: 0;
-                  width: 0;
-                  height: 2px;
-                  background: #0056EC;
-                  opacity: 0;
-                  transition: opacity .5s, width .5s;
+  position: absolute;
+  display: block;
+  content: '';
+  left: 0;
+  width: 0;
+  height: 2px;
+  background: #0056EC;
+  opacity: 0;
+  transition: opacity .5s, width .5s;
 }
 
 .menu__item:hover {
-                  opacity: 0.5;
+  opacity: 0.5;
 }
 
 .menu__item:hover:after, .menu__item_active:after {
-                                                  opacity: 1;
-                                                  width: 100%;
+  opacity: 1;
+  width: 100%;
+}
+
+.main {
+  padding: 0 63% 200px 0;
+  background: no-repeat top right url(img/mainImg.svg);
+}
+
+.main__title {
+  padding-top: 70px;
+  margin: 0 0 40px 0;
+  font: bold 66px/87px pt_serifbold, arial, helvetica, sans-serif;
+}
+
+.main__title-text_highlight {
+  background: #ECF3FF;
```

Comment:
Codacy found an issue: [Expected "#ECF3FF" to be "#ecf3ff" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/IrirniProject/pullRequest?prid=6241594)